src/Time/TimerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Time;
+
+interface TimerInterface
```

Comment:
There will be soon an interface for it: https://github.com/php-fig/fig-standards/blob/master/proposed/clock.md

Reply:
https://github.com/cseufert/clock

Reply:
```javascript
"repositories": [
    {
      "type": "github",
      "url": "https://github.com/cseufert/clock"
    }
  ],
```

Reply:
Nope, that won't work because of microtime accuracy.